app/assets/javascripts/editor/participantsupport.js

```diff
@@ -1,5 +1,5 @@
 CodeOceanEditorFlowr = {
-  isFlowrEnabled: <%= CodeOcean::Config.new(:code_ocean).read[:flowr][:enabled] %>,
+  isFlowrEnabled: document.documentElement.dataset.flowrEnabled === 'true',
```

Comment:
As I mentioned already in another PR, we don't have a linter for JS and CSS yet. Still, some common styles are appreciated.

In this file, you used `'true'`. In `app/assets/javascripts/editor/editor.js.erb`, you used `"true"`.